lib/tapioca/compilers/dsl/url_helpers.rb

```diff
@@ -0,0 +1,82 @@
+# typed: true
+# frozen_string_literal: true
+
+require "parlour"
+
+begin
+  require "rails"
+rescue LoadError
+  return
+end
+
+module Tapioca
+  module Compilers
+    module Dsl
+      class UrlHelpers < Base
+        extend T::Sig
+
+        sig { override.params(root: Parlour::RbiGenerator::Namespace, constant: T.class_of(Object)).void }
+        def decorate(root, constant)
+          named_routes = Rails.application.routes.named_routes
+          path_helper_methods = named_routes.path_helpers_module.instance_methods(false)
```

Comment:
This can be just:
```suggestion
          path_helper_methods = GeneratedPathHelpersModule.instance_methods(false)
```
now